README.md

```diff
@@ -1,284 +1,126 @@
 
-```typescript
-// ./pages/ssr.tsx
+### Install next-on-pages and vercel (optional)
 
-import type { NextPage } from 'next';
-import Head from 'next/head';
-import styles from '../styles/Home.module.css';
+To speed up local development (especially the refresh speed in watch mode) you can install next-on-pages and `vercel` as dev dependencies of your project, note that this is optional and if you decide to install only one of them or neither everything will still run correctly anyways:
 
-export const config = {
-	runtime: 'experimental-edge',
-};
+```sh
+npm install -D @cloudflare/next-on-pages vercel
+```
 
-export const getServerSideProps = async () => {
-	return {
-		props: {
-			runtime: process.env.NEXT_RUNTIME,
-			uuid: await fetch('https://uuid.rocks/plain').then(response =>
-				response.text()
-			),
-		},
-	};
-};
+## Examples
 
-const Home: NextPage<{ runtime: string; uuid: string }> = ({
-	runtime,
-	uuid,
-}) => {
-	return (
-		<div className={styles.container}>
-			<Head>
-				<title>Create Next App</title>
-				<meta name="description" content="Generated by create next app" />
-				<link rel="icon" href="/favicon.ico" />
-			</Head>
-
-			<main className={styles.main}>
-				<h1 className={styles.title}>
-					Welcome to{' '}
-					<a href="https://nextjs.org">Next.js, running at the {runtime}!</a>
-				</h1>
-
-				<p className={styles.description}>
-					Get started by editing{' '}
-					<code className={styles.code}>pages/index.tsx</code>
-				</p>
-
-				<p className={styles.description}>
-					Here&apos;s a server-side UUID:
-					<code className={styles.code}>{uuid}</code>
-				</p>
-			</main>
-		</div>
-	);
-};
+To see some examples on how to use Next.js features with next-on-pages see the [Examples document](./docs/examples.md).
 
-export default Home;
-```
+## Contributing
 
-### [Middleware](https://nextjs.org/docs/advanced-features/middleware)
+If you want to contribute to the project please refer to the [Contributing document](./docs/contributing.md).
 
-```typescript
-import { NextResponse } from 'next/server';
-import type { NextRequest } from 'next/server';
+## References
 
-// This function can be marked `async` if using `await` inside
-export function middleware(request: NextRequest) {
-	return NextResponse.redirect(new URL('/about-2', request.url));
-}
+Extra references you might be interested in:
 
-export const config = {
-	matcher: '/about/:path*',
-};
-```
+- [Blog post](https://blog.cloudflare.com/next-on-pages)
+
+  The original blog post introducing next-on-pages (24/10/2022), it goes into details on how the package came to be and provides details on how it works.
+
+- [Cloudflare Next.js Guide](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/)
+
+  Cloudflare guide on how to create and deploy a Next.js application. The application can be either static (and deployed as simple static assets) or dynamic using the edge runtime (via next-on-pages).
```

Comment:
```suggestion
  Cloudflare guide on how to create and deploy a Next.js application. The application can be either static (and deployed as simple static assets) or dynamic using the Edge Runtime (powered by `@cloudflare/next-on-pages`).
```

Reply:
Mh... next-on-pages isn't powering the edge runtime, is it? 🤔